metrics.go

```diff
@@ -75,15 +70,14 @@ func NewExpVarMetrics(publish bool) *ExpVarMetrics {
 		return new(expvar.Map).Init()
 	}
 	return &ExpVarMetrics{
-		LockContentionVar:            newMap("lock_contention"),
-		GraveyardCleaningDurationVar: newMap("graveyard_cleaning_duration"),
-		GraveyardLowWatermarkVar:     newMap("graveyard_low_watermark"),
-		GraveyardObjectCountVar:      newMap("graveyard_object_count"),
-		ObjectCountVar:               newMap("object_count"),
-		WriteTxnAcquisitionVar:       newMap("write_txn_acquisition"),
-		WriteTxnDurationVar:          newMap("write_txn_duration"),
-		DeleteTrackerCountVar:        newMap("delete_tracker_count"),
-		RevisionVar:                  newMap("revision"),
+		LockContentionVar:        newMap("lock_contention"),
+		GraveyardLowWatermarkVar: newMap("graveyard_low_watermark"),
```

Comment:
I think keeping the graveyard naming makes sense here as `deleted_object_count` would be rather confusing.

Reply:
I guess it would be `objects_pending_deletion_count`, but yeah, graveyard here is more concise.